resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentCredentialServiceImpl.java

```diff
@@ -320,7 +320,11 @@ private ResidentTransactionEntity createResidentTransactionEntity(ResidentCreden
 		residentTransactionEntity.setTokenId(identityServiceImpl.getResidentIdaToken());
 		residentTransactionEntity.setAuthTypeCode(identityServiceImpl.getResidentAuthenticationMode());
 		residentTransactionEntity.setRequestSummary(EventStatusSuccess.DATA_SHARED_SUCCESSFULLY.name());
-		String attributeList = dto.getSharableAttributes().stream().collect(Collectors.joining(", "));
+		List<String> sharableAttributes = dto.getSharableAttributes();
+		if(sharableAttributes == null){
```

Comment:
Add missing input parameter error for sharable attributes if it is null or empty.

Reply:
Done.